CHANGELOG.md

```diff
@@ -121,6 +121,7 @@ multi-echo aquisitions (PR #7515).
 from the `psc` table. The default value of `CenterID` is `NULL`. Previously, the 
 default for `Center_name` was `AAAA` or `ZZZZ`. (PR #7525)
 - The statistics displayed in the dashboard was changed to only show the data relevant to the user's site(s). (PR #8132)
+- BVL Feedback widget only shows notifications for the users sites / projects (PR #7848)
```

Comment:
This is in the wrong release in the CHANGELOG. 24 is already released and this PR is going to main..

Reply:
Fixed!